YvesIraguha/TeaHouse
# #170046534 Prepare Deploying Application

#### What does this PR do?
-  Prepare scripts to deploy the application
#### Description of Task to be completed?
- Set up Heroku pipeline
#### How should this be manually tested?
- Clone the repo and checkout the branch `deploy-application-on-heroku-170046534`
- Run `npm i && npm run precommit ` to run tests 
- Make changes to the codebase, push changes, and create a pull request. 
- The application should be immediately deployed. 

#### What are the relevant pivotal tracker stories?
#[170046534](https://www.pivotaltracker.com/story/show/170046534)
